db/seeds.rb

```diff
@@ -57,13 +61,33 @@ def seed_namespace_group_links(user, namespace, group, group_access_level)
                                      expires_at: @namespace_group_link_expiry_date }).execute
 end
 
-def seed_samples(project, sample_count)
+def fake_metadata
+  {
+    'WGS_id' => Faker::Number.number(digits: 10),
+    'NCBI_ACCESSION' => "NM_#{Faker::Number.decimal(l_digits: 7, r_digits: 1)}",
+    'country' => Faker::Address.country,
+    'food' => Faker::Food.dish,
+    'gender' => Faker::Gender.binary_type,
+    'age' => Faker::Number.between(from: 1, to: 100),
+    'onset' => Faker::Date.between(from: 2.years.ago.to_s, to: Time.zone.today),
+    'earliest_date' => Faker::Date.between(from: 2.years.ago.to_s, to: Time.zone.today),
+    'patient_sex' => Faker::Gender.binary_type
+    # SourceState	species	subspecies	serovar	patient_sex	patient_age
```

Comment:
Remove these commented out lines unless there's a good reason of why they are there

Reply:
Updated in eb24a15